lib/middleware/job_test.go

```diff
@@ -25,7 +26,7 @@ func TestBuildRundeckURLEndpoint(t *testing.T) {
 func TestGetRundeckJobID(t *testing.T) {
 	// mock api call
 	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		w.Header().Set("Content-Type", "application/json")
+		w.Header().Set("Accept", "application/json")
```

Comment:
FYI `Content-Type` header as `application/json` is actually more important here as it serves to inform the client receiving the response the type of the data the server sent. So you had it right before.
But this is trivial since the implementation indiscriminately executes `json.Unmarshal`

Reply:
will switch to `"Content-Type"` to make all tests consistent